### PR TITLE
UD-894: set correct state on "Back to IDE" button

### PR DESCRIPTION
**Pull request by akurinnoy**

### CODE REVIEW

**Comment by benoitf** [comment]
+1

**Comment by benoitf** [comment]
It works like this but I think that we could have also track change of route in navbar component and reset the select mode if the route is not part of the navbar component

Then everything was tied to navbar

**Comment by akurinnoy** [comment]
@ashumilova  @benoitf WDYT

**Comment by benoitf** [comment]
look very nice but I don't know if we merge now or wait beta release

**Comment by ashumilova** [comment]
+1 (with small changes) and keep waiting for beta release

File: src/app/workspaces/create-workspace/create-workspace.controller.js [hunk 1]
```diff
@@ -50,8 +50,6 @@ class CreateWorkspaceCtrl {
     promise.then(() => {
       this.updateData();
     });
-
```
Comment (benoitf) [inline src/app/workspaces/create-workspace/create-workspace.controller.js]: you should remove rootScope from the constructor as it was introduced for this broadcast

**Comment by akurinnoy** [comment]
@benoitf I've removed that unused $rootScope

**Comment by benoitf** [comment]
+1 very nice

**Pull request merged**